src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
```

Comment:
I believe this test would fail.
see if you can:
1. run the test
2. verify that it is failing
3. change the test
4. (bonus): make it to test the component *functionality* (test task addition and deletion)

Reply:
never worked on the tests tho...
those one are in default

Reply:
I know that,
see it as an opportunity